start.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+docker build -t 3mdeb/pxe-server .
+
+if [ $? -ne 0 ]; then
+    echo "ERROR: Unable to build container"
+    exit 1
+fi
+
+
+docker run --name pxeserver --privileged \
+	 -p 111:111/tcp -p 69:69/udp -p 2049:2049/tcp \
+	 -p 627:627/tcp -p 627:627/udp -p 875:875/tcp -p 875:875/udp \
+	 -p 892:892/tcp -p 892:892/udp -p 111:111/udp -p 2049:2049/udp \
+	 -p 10053:10053/udp -p 10053:10053/tcp \
+	 -p 32769:32769/tcp -p 32769:32769/udp \
+	 -p 32765:32765/tcp -p 32765:32765/udp \
+	 -p 32766:32766/tcp -p 32766:32766/udp \
+	 -p 32767:32767/tcp -p 32767:32767/udp \
+	 -v ${PWD}/netboot:/srv/tftp \
+	 -v ${PWD}/debian:/srv/nfs/debian \
+	 -v ${PWD}/voyage:/srv/nfs/voyage \
+	 -t -i 3mdeb/pxe-server /bin/bash -c \
+	 "bash /usr/local/bin/run.sh;/bin/bash" $@
```

Comment:
@miczyg1 of course no check if above ports are in use on host:

```
docker: Error response from daemon: driver failed programming external connectivity on endpoint pxeserver (607f43aaa8e1c02ed6c5d182433582079b7703f5fa2c3d71ff3b92956b0807b3): Error starting userland proxy: listen tcp 0.0.0.0:2049: bind: address already in use.
```

Reply:
@miczyg1 stopping `nfs-kernel-server` is not enough. I had to uninstall `rpcbind` to get rid of 111. This solution is really hairy.

Reply:
@miczyg1 also please add `--rm` to avoid issues when container will be run next time.